frontend/src/Components/ClientProfileContent.js

```diff
@@ -338,39 +345,41 @@ class ClientProfileContent extends Component {
       }
       return (
         <React.Fragment>
-          <div>
-            <InputComponent
-              type='text'
-              label='Short Term Goal (Examples: taking a vacation, buying a car)'
-              placeholder='Type in your short term goal'
-              value={props.shorttermgoal}
-              name='shorttermgoal'
-              onChange={props.handleChange}
-              required
-            />
-          </div>
-          {/* TODO: Add inputs on button click */}
-          <Center>
-            <ToastButtonComponent tertiary label='+ Short Term Goal' />
-          </Center>
 
-          <hr />
+          <InputComponent type='text' name='goal' label='Goal 1' list='goals' placeholder='Type in a goal...' value={props.goal} onChange={props.handleChange} />
```

Comment:
small detail but for consistency with placeholder text, remove the '...' at the end. and use 'your' instead of 'a'